resources/views/api/menu/login_logout.blade.php

```diff
@@ -1,13 +1,59 @@
-@auth
-    @component('components.navs.nav_element', [
+@component('components.navs.nav_element', ['contentClass' => 'small'])
+    @slot('options')
+        style="cursor: default"
+    @endslot
+    v{{ config('app.version') }}
+@endcomponent
+@component('components.navs.nav_element', [
         'route' => route('web.user.dashboard')
     ])
+    <div class="btn btn-secondary" style="margin: 0;">
+        @component('components.elements.icon', ['class' => 'mr-1'])
+            tachometer-alt
+        @endcomponent
         @lang('Dashboard')
-    @endcomponent
+    </div>
+@endcomponent
+@component('components.navs.nav_element')
+    <div class="nav-item dropdown">
+        <button class="btn btn-secondary dropdown-toggle" type="button" data-toggle="dropdown" aria-expanded="false">
+            @component('components.elements.icon', ['class' => 'mr-1'])
+                palette
+            @endcomponent
+            Themes
+        </button>
+        <div class="dropdown-menu">
+            <a class="dropdown-item" href="#">
+                @component('components.elements.icon', ['class' => 'mr-1'])
+                    moon
+                @endcomponent
+                Darkmode
+            </a>
+            <div class="dropdown-divider"></div>
+            <a class="dropdown-item" href="#">
+                @component('components.elements.icon', ['class' => 'mr-1'])
+                    sun
+                @endcomponent
+                Whitemode
+            </a>
+            <div class="dropdown-divider"></div>
+            <a class="dropdown-item" href="#">
+                @component('components.elements.icon', ['class' => 'mr-1'])
+                    user-astronaut
+                @endcomponent
+                SpaceTheme (Recolor)
```

Comment:
I'd change `Whitemode` to `Lightmode` and add translations for those keys, i.e. `@lang('Darkmode')`.  
For the german translation you could use something simple like `Dunkel` and `Hell`